src/main/java/org/vafer/jdeb/utils/MapVariableResolver.java

```diff
@@ -34,4 +40,96 @@ public String get( String key ) {
         return map.get(key);
     }
 
+    public Map<String, String> getMap() {
+        return map;
```

Comment:
Is returning the full Map really required?
If possible I would prefer to keep the interface as stupid as possible.

Reply:
🤔 I added it to simplify testing. I could remove it and proceed via `get`s if you prefer

Reply:
Removing it would be cleaner - but I could also live with just comment stating it should only be used for tests.

Reply:
Addressed in 8d08c3c55c3e46bbec83cbb370acc8198203a33a